tests/data/test_cc_oop/report_309ac2fd7f2dcf17.txt

```diff
@@ -1,16 +1,16 @@
-�rendetyp: 6 Diarienummer: 18FMV7705-43:1
+Ärendetyp: 6 Diarienummer: 18FMV7705-43:1
```

Comment:
Why did the id in this filename change? Is it the `dgst` of the certificate? If so, wy the change? Shouldn't that be static?